recipe/meta.yaml

```
@@ -1,32 +1,32 @@
 {% set name = "greenlet" %}
-{% set version = "3.1.1" %}
+{% set version = "3.2.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/greenlet-{{ version }}.tar.gz
-  sha256: 4ce3ac6cdb6adf7946475d7ef31777c26d94bccc377e070a7986bd2d5c515467
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0dca0d95ff849f9a364385f36ab49f50065d76964944638be9691e1832e9f86d
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv --ignore-installed .
```

https://github.com/python-greenlet/greenlet/blob/3.2.4/setup.py#L266
```suggestion
  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv --ignore-installed .
  skip: True  # [py<39]
```